cluster/gce/gci/configure-helper.sh

```diff
@@ -1954,6 +1954,26 @@ function setup-addon-manifests {
   fi
 }
 
+# A function that downloads extra addons from a URL and puts them in the GCI
+# manifests directory.
+function download-extra-addons {
+  local -r out_dir="${KUBE_HOME}/kube-manifests/kubernetes/gci-trusty/gce-extras"
+
+  mkdir -p "${out_dir}"
+
+  local curl_cmd=("curl" "--fail" "--retry=5" "--retry-delay=3" "--silent" "--show-error")
+  if curl --help | grep -q -- '--retry-connrefused'; then
```

Comment:
This check is already done at the top of the file. You can use `${CURL_RETRY_CONNREFUSED}` to get the flag if it's supported without redoing the check here.